### PR TITLE
feat: add a package for Vue components library

### PR DESCRIPTION
**Pull request by lauthieb**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Make sure you are making a pull request against the **main branch** (left side).
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Description

Add a Vue components library.

## Does this introduce a breaking change?

- No

❤️ Thank you!

### CODE REVIEW

File: packages/showcases/vue/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "license": "Apache-2.0",
+  "scripts": {
+    "build-storybook": "build-storybook",
+    "build-storybook:docs": "build-storybook --docs",
+    "st": "yarn start",
+    "start": "start-storybook -p 6010"
+  },
+  "dependencies": {
+    "core-js": "^3.6.5",
+    "vue": "^3.0.0"
```
Comment (gabbloquet) [inline packages/showcases/vue/package.json:15]: 💥

**Review by AgatheGossey — APPROVED** [review]

File: packages/showcases/vue/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "@storybook/addon-essentials": "6.2.0-alpha.24",
+    "@storybook/addon-links": "6.2.0-alpha.24",
+    "@storybook/vue3": "6.2.0-alpha.24",
+    "@vtmn/vue": "*",
+    "@vue/cli-plugin-babel": "~4.5.0",
+    "@vue/cli-plugin-typescript": "~4.5.0",
+    "@vue/cli-service": "~4.5.0",
+    "@vue/compiler-sfc": "^3.0.0",
+    "babel-loader": "^8.2.2",
+    "typescript": "~3.9.3",
```
Comment (Anthony-dkt) [inline packages/showcases/vue/package.json:29]: Typescript can be upgraded to latest major version ( 4.1.3 )

File: packages/sources/vue/package.json [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    "postcss-simple-vars": "5.0.2",
+    "postcss-url": "^8.0.0",
+    "rimraf": "^3.0.2",
+    "rollup": "^2.30.0",
+    "rollup-plugin-css-only": "^2.1.0",
+    "rollup-plugin-postcss": "^3.1.8",
+    "rollup-plugin-terser": "^7.0.2",
+    "rollup-plugin-typescript2": "0.28.0",
+    "rollup-plugin-vue": "^6.0.0-beta.10",
+    "typescript": "3.9.3",
```
Comment (Anthony-dkt) [inline packages/sources/vue/package.json:85]: Same comment as before (latest version ? )

Comment (lauthieb) [reply packages/sources/vue/package.json]: done :)

File: packages/showcases/vue/package.json (hunk 2 above)
Comment (lauthieb) [reply packages/showcases/vue/package.json]: done :)

**Review by Anthony-dkt — APPROVED** [review]

**Comment by lauthieb** [comment]
We still have a problem with the rest props. How do we pass all the rest props to the component? Because here I only pass “variant” and “size” props, for example, if you add “onclick” it will not be passed through the `<button>`. With Svelte & React I succeed, but I don’t know how to do this with Vue 3 with reactive props.

**Review by Z26PGOBL — APPROVED** [review]

**Pull request merged**